lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1268,6 +1268,12 @@ static const char *getImportFailureString(swift::serialization::Status status) {
   case swift::serialization::Status::TargetTooNew:
     return "The module file was built for a target newer than the current "
            "target.";
+  case swift::serialization::Status::SDKMismatch:
+    return "The module file was built with a different SDK than the one in use "
+           "to build the client.";
+  case swift::serialization::Status::RevisionIncompatible:
+    return "The resilient module file was built by a different version of the "
```

Comment:
Is "resilient" part of the wording in the header docs here or something we came up with? Users might confuse it with resilient types (aka library evolution). If it's not important, I would just use "module file" without qualifiers here.

Reply:
This check is only applied to modules generated with `-enable-library-evolution`. There are other version related error states (`FormatToo{Old,New}`), and I thought it might be useful to explicitly distinguish this from the others. For context, this check is in https://github.com/apple/swift/pull/39304.

Reply:
How about "The module file has library evolution enabled and was built by a different version of the compiler." ?

Reply:
I went with "The module file was built with library evolution enabled by a different version of the compiler."